CSS/style.css

```diff
@@ -0,0 +1,903 @@
+body {
+  background-color: #f4f5f7;
+  width: 100vw;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  width: 100vw;
+  position: relative;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  background-color: white;
+}
+
+#logo {
+  text-decoration: none;
+  color: #6070ff;
+  margin: 24px auto 24px 20px;
+  width: 100vm;
+  height: 20px;
+  font-size: 18px;
+  display: flex;
+  font-weight: 700;
+}
+
+p,
+a {
+  font-size: larger;
+}
+
+h1,
+h2 {
+  font-size: x-large;
+}
+
+.nav-pic {
+  height: 12px;
+  padding-right: 15px;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 15px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+form li {
+  margin-bottom: 20px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+  margin-left: 32px;
+}
+
+.nav-ul a {
+  text-decoration: none;
+  transition: transform 0.2s;
+}
+
+.home {
+  background-image: url("../img/background.png");
+  background-color: white;
+  background-size: cover;
+  border-bottom-left-radius: 70px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 50px 24px;
+  padding-right: 0;
+}
+
+.home h1 {
+  font-size: 40px;
+  margin-bottom: 0;
+  font-weight: 900;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.home p {
+  width: 270px;
+  font-size: 16px;
+}
+
+.home-content {
+  /* margin: 0% 0% 0% 8%; */
+  color: #172b4d;
+}
+
+.home-content p {
+  color: #172b4d;
+}
+
+.connect p {
+  color: #7f8cff;
+}
+
+.home li {
+  padding-right: 20px;
+}
+
+form > ul {
+  list-style: none;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.home > ul {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+}
+
+.work {
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  left: 0;
+  top: 100px;
+}
+
+.work.active {
+  filter: blur(50px);
+}
+
+.grid-item {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  padding: 15px 5px 15px 15px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 90 %;
+  margin: auto;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 0;
+  margin-top: 12px;
+}
+
+.work-content {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 24px;
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- [ ]  **[Optional]** Kindly, you may try to check the font-size and try to replicate the Figma design as much as possible 🙏

![image](https://user-images.githubusercontent.com/102757126/227742381-ae295815-5454-413a-acce-51e8587ffc29.png)